admin/src/main/resources/io/buoyant/admin/js/src/admin.js

```diff
@@ -19,17 +19,16 @@ define([
 
     return $.get("config.json").done(function(routersRsp) {
       var template = Handlebars.compile(templateRsp);
-      var routers = routersRsp.routers;
+      var routers = _.sortBy(routersRsp.routers, "label");
```

Comment:
prior to sorting here, are the routers in an arbitrary order, or the order they appear in the config file?  Preserving the order from the config file might be nice.